packages/react-core/src/components/Card/Card.tsx

```diff
@@ -139,24 +104,9 @@ export const Card: React.FunctionComponent<CardProps> = ({
   const containsCardTitleChildRef = React.useRef(false);
 
   const registerTitleId = (id: string) => {
-    setTitleId(id);
     containsCardTitleChildRef.current = !!id;
   };
```

Comment:
I believe we should be able to remove this `registerTitleId` here and from the context (and its use within CardTitle), as well as the `containsCardTitleChildRef` ref. It was really only being used to get the card title ID from CardTitle to set the aria-labelledby attribute in Card.

In CardTitle we should be able to remove the useEffect on line ~25, too.